proxy/envoy/testdata/lds-v0-none-auth-optout.json.golden

```diff
@@ -0,0 +1,624 @@
+{
+  "listeners": [
+   {
+    "address": "tcp://0.0.0.0:15001",
+    "name": "virtual",
+    "filters": [],
+    "bind_to_port": true,
+    "use_original_dst": true
+   },
+   {
+    "address": "tcp://0.0.0.0:443",
+    "name": "http_0.0.0.0_443",
+    "filters": [
+     {
+      "type": "read",
+      "name": "http_connection_manager",
+      "config": {
+       "codec_type": "auto",
+       "stat_prefix": "http",
+       "generate_request_id": true,
+       "tracing": {
+        "operation_name": "egress"
+       },
+       "rds": {
+        "cluster": "rds",
+        "route_config_name": "443",
+        "refresh_delay_ms": 10
+       },
+       "filters": [
+        {
+         "type": "decoder",
+         "name": "mixer",
+         "config": {
+          "mixer_attributes": {
+           "destination.ip": "10.1.1.0",
+           "destination.service": "hello.default.svc.cluster.local",
+           "destination.uid": "kubernetes://v0.default"
+          },
+          "forward_attributes": {
+           "source.ip": "10.1.1.0",
+           "source.uid": "kubernetes://v0.default"
+          },
+          "quota_name": "RequestCount"
+         }
+        },
+        {
+         "type": "decoder",
+         "name": "router",
+         "config": {}
+        }
+       ],
+       "access_log": [
+        {
+         "path": "/dev/stdout"
+        }
+       ]
+      }
+     }
+    ],
+    "bind_to_port": false
```

Comment:
where is the SSL context for this listener?

Reply:
This listener is for outgoing traffic right? ssl_context is added for inbound listener only.  This golden file is branched from lds-v0-none-auth. FYI, here is the diff:

diff proxy/envoy/testdata/lds-v0-none-auth-optout.json.golden proxy/envoy/testdata/lds-v0-none-auth.json.golden 

```
523a524,529
>     "ssl_context": {
>      "cert_chain_file": "/etc/certs/cert-chain.pem",
>      "private_key_file": "/etc/certs/key.pem",
>      "ca_cert_file": "/etc/certs/root-cert.pem",
>      "require_client_certificate": true
>     },
624c630
<  }
---
>  }
\ No newline at end of file
```